src/bridge/bridge.c

```diff
@@ -391,17 +394,39 @@ start_ssh_agent (void)
       goto out;
     }
 
-  flags = G_SPAWN_SEARCH_PATH | G_SPAWN_STDOUT_TO_DEV_NULL;
-  g_spawn_async (NULL, agent_argv, NULL, flags,
-                 setup_ssh_agent, GINT_TO_POINTER (-1), &pid, &error);
-
-  if (error != NULL)
+  if (!g_spawn_sync (NULL, agent_argv, NULL,
+                     G_SPAWN_SEARCH_PATH, setup_ssh_agent,
+                     GINT_TO_POINTER (-1),
+                     &agent_output, &agent_error,
```

Comment:
I don't think g_spawn_sync supports that, you either supply a pointer or use G_SPAWN_STDERR_TO_DEV_NULL